standup/status/migrations/0006_add_irc_nick.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('status', '0005_add_team_user_id'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='standupuser',
+            name='irc_nick',
+            field=models.CharField(help_text='IRC nick for this particular user',
+                                   max_length=100, null=True, unique=True, blank=True),
+        ),
+        migrations.AlterField(
```

Comment:
Not sure why `github_handle` and `slug` are being modified in the migration. They don't seem to have been touched in models.py.